bokeh/properties.py

```diff
@@ -124,13 +124,14 @@ class DeserializationError(Exception):
 class Property(object):
     """ Base class for all type properties. """
 
-    def __init__(self, default=None, help=None):
+    def __init__(self, default=None, help=None, serialized=True):
```

Comment:
So, the default is `True` and later you change this with `prop._serialized = False`, so maybe this argument is unnecessary? If it's left, then specific property classes should be allowed to get this argument as well, so you would have to change a lot of constructors (won't be pretty).